### PR TITLE
Bump micronaut-test to 3.2.0

### PR DESCRIPTION
**Pull request by micronaut-build**

### CODE REVIEW

**Comment by sdelamo** [comment]
@timyates can you investigate why this is failing?

**Comment by timyates** [comment]
@sdelamo Please eyeball my changes to ensure I did good 😀👍

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -407,6 +408,8 @@ junit-jupiter-engine = { module = "org.junit.jupiter:junit-jupiter-engine", vers
 junit-vintage = { module = "org.junit.vintage:junit-vintage-engine", version.ref = "managed-junit5" }
 junit4 = { module = "junit:junit", version.ref = "junit4" }
 
+jetbrains-annotations = { module = "org.jetbrains:annotations", version.ref = "jetbrains-annotations" }
```
Comment (sdelamo) [inline gradle/libs.versions.toml:408]: remove

File: http-server-netty/build.gradle [hunk 2]
```diff
@@ -31,6 +31,7 @@ dependencies {
     compileOnly libs.kotlin.stdlib
 
     testCompileOnly project(":inject-groovy")
+    testCompileOnly(libs.jetbrains.annotations)
```
Comment (sdelamo) [inline http-server-netty/build.gradle:34]: remove

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -16,6 +16,7 @@ jakarta-inject-api = "2.0.1"
 jakarta-inject-tck = "2.0.1"
 javax-inject = "1"
 javax-persistence = "2.2"
+jetbrains-annotations = "23.0.0"
```
Comment (sdelamo) [inline gradle/libs.versions.toml:19]: remove

**Review by sdelamo — CHANGES_REQUESTED** [review]

File: gradle/libs.versions.toml (hunk 1 above)
Comment (timyates) [reply gradle/libs.versions.toml:408]: So you don't want the `@Language` annotation available here?

https://github.com/micronaut-projects/micronaut-core/blob/019c370f4db0e0aa5517adc8fd8ce4213b8efdb0/http-server-netty/src/test/groovy/io/micronaut/http/server/netty/http2/Http2ServerPushSpec.groovy#L266-L274

Comment (timyates) [reply gradle/libs.versions.toml:408]: (That's why I added it)

Comment (sdelamo) [reply gradle/libs.versions.toml:408]: I saw that. it is fine. I have removed the rest of the NotNulls https://github.com/micronaut-projects/micronaut-core/pull/7351/commits/0ea1302a9ec7d9ef557b16253f3e3c7aa195bcb1

**Review by sdelamo — APPROVED** [review]

**Review by sdelamo — APPROVED** [review]

**Pull request merged**